javaagent-tooling/src/test/groovy/io/opentelemetry/javaagent/tooling/matcher/AdditionalLibraryIgnoresMatcherTest.groovy

```diff
@@ -0,0 +1,60 @@
+package io.opentelemetry.javaagent.tooling.matcher
+
+import net.bytebuddy.description.type.TypeDescription
+import spock.lang.Specification
+
+class AdditionalLibraryIgnoresMatcherTest extends Specification {
+
+  def underTest = new AdditionalLibraryIgnoresMatcher();
+
+  def "spring boot - match not instrumented class"() {
+
+    setup:
+    def type = Mock(TypeDescription)
+    type.getActualName() >> typeName
+
+    when:
+    def matches = underTest.matches(type)
+
+    then:
+    matches == true
```

Comment:
You can use `expect:` instead

```suggestion
    expect:
    underTest.matches(type)
```

Reply:
I like to stick to GWT BDD test convention.